sdk/src/connections/target/addi9036_sensor.cpp

```diff
@@ -458,6 +461,11 @@ aditof::Status Addi9036Sensor::program(const uint8_t *firmware, size_t size) {
     static unsigned char buf[CTRL_PACKET_SIZE];
     size_t readBytes = 0;
 
+    if (firmware == nullptr) {
+        LOG(WARNING) << "Invalid firmware";
```

Comment:
- this should be an `ERROR`
- It`s not the firmware that is invalid, but the pointer.